projects/uitdatabank/docs/search-api/sorting.md

```diff
@@ -0,0 +1,86 @@
+# Sorting
+
+The UiTdatabank Search API provides several options to sort events, places and organizers.
+
+- `availableTo` 
+- `created`
+- `distance` 
+- `modified`
+- `score`
+
+For every sort parameter it is possible to change the sort `order`:
+
+- Ascending order \(from smallest to biggest value\) by using `asc`
+- Descending order \(from biggest to smallest value\) by using `desc`
+
+## Sort parameters
+### availableTo
+<!-- theme: info -->
+
+> Only applicable on events and places.
+
+Sorts the data by the date until it is available in the API. This is most commonly used to show events & places that will be ending soon first. Sorting on availableTo (ascending) is what we advice for event calendars.
+
+**Example**
+```
+GET https://search.uitdatabank.be/offers/?sort[availableTo]=asc
+```
+Events & places that end soon will appear first, permanent events and places last.
+
+### created
+Sorts the data by the date it is created in the API
```

Comment:
```suggestion
Sorts the data by the date it is created in the API.
```

(Every other sentence in the guide ends with a period, so I assume it's missing here by accident and not intentional)